ComputeDemo/ComputeAlphaMainIntegrationTest.cs

```diff
@@ -0,0 +1,115 @@
+        private readonly string _region;
+        private readonly AddressesClient _addrClient;
+        private readonly RegionOperationsClient _ropsClient;
+
+        const string CredentialsPath = @"C:\Users\virost\AppData\Roaming\gcloud\legacy_credentials\virost@google.com\adc.json";
+
+        public ComputeAlphaMainIntegrationTest(ITestOutputHelper output)
+        {
+            _output = output;
+            _project = "client-debugging";
+            _region = "us-central1";
+            
+            _addrClient = new AddressesClientBuilder
+            {
+                CredentialsPath = CredentialsPath
+            }.Build();
+
+            _ropsClient = new RegionOperationsClientBuilder
+            {
+                CredentialsPath = CredentialsPath
+            }.Build();
+        }
+
+        [xunit::Fact]
+        public void CreateDeleteIPIntegration()
+        {
+            AddressesClient addressesClient = _addrClient;
+            var addrName = $"testaddr-csharp-{Guid.NewGuid()}";
+
+            _output.WriteLine($"Retrieving address with the name: {addrName}");
+
+            var exc = xunit.Assert.ThrowsAny<grpccore.RpcException>(
+                () => addressesClient.Get(_project, _region, addrName)
+            );
+            xunit.Assert.Equal(grpccore.StatusCode.NotFound, exc.StatusCode);
+
+            Address addressResource = new Address
+            {
+                Name = addrName,
+                Region = _region,
+                NetworkTier = Address.Types.NetworkTier.Premium
+            };
+
+            _output.WriteLine($"Creating address with the name: {addrName}");
+            Operation insertOp = addressesClient.Insert(_project, _region, addressResource);
+            _output.WriteLine($"Operation to create address: {insertOp.Name} status {insertOp.Status}; start time {insertOp.StartTime}");
+            insertOp = PollForCompletion(insertOp, "create");
+            _output.WriteLine($"Operation to create address completed: status {insertOp.Status}; start time {insertOp.StartTime}; end time {insertOp.EndTime}");
+
+            _output.WriteLine($"Retrieving address with the name: {addrName}");
+            Address readAddr = addressesClient.Get(_project, _region, addrName);
+            xunit.Assert.NotNull(readAddr);
+            xunit.Assert.Equal(readAddr.Name, addrName);
+
+            _output.WriteLine($"Deleting address with the name: {addrName}");
+            Operation deleteOp = addressesClient.Delete(_project, _region, addrName);
+            _output.WriteLine($"Operation to delete address: {deleteOp.Name} status {deleteOp.Status}; start time {deleteOp.StartTime}");
+            deleteOp = PollForCompletion(deleteOp, "delete");
+            _output.WriteLine($"Operation to delete address completed: status {deleteOp.Status}; start time {deleteOp.StartTime}; end time {deleteOp.EndTime}");
+        }
+
+        private Operation PollForCompletion(Operation insertOp, string alias)
```

Comment:
I can't remember whether we're planning on doing better than this for DIREGAPIC. It would be nice to use the existing `Operation<TResult, TMetadata>` class, but I can see that may not be feasible.

Let's talk about plans when we meet.